test/extended/networking.sh

```diff
@@ -7,9 +7,19 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-if [[ "${OPENSHIFT_QUIET_OUTPUT:-false}" != "true" ]]; then
+CONFIG_ROOT="${OPENSHIFT_CONFIG_ROOT:-}"
+
+# CI mode (deploying clusters for each plugin) should be noisy by
+# default to aid in debugging.
+if [[ "${CONFIG_ROOT}" = "" ]]; then
+  DEFAULT_QUIET=false
+else
+  DEFAULT_QUIET=true
+fi
+
+if [[ "${OPENSHIFT_QUIET_OUTPUT:-${DEFAULT_QUIET}}" != "true" ]]; then
```

Comment:
The code above allows a developer to pass some non-null string in using *either* `$OPENSHIFT_QUIET_OUTPUT` or `$OPENSHIFT_CONFIG_ROOT` -- I believe this is an equivalent logical statement to what you had written, except for the fact that the *content* of `$OPENSHIFT_QUIET_OUTPUT` is no longer important, simply the presence of some non-null string is `true` -- that's idiomatic for Bash.

For instance:
```sh
function existing_logic() {
	local CONFIG_ROOT="${OPENSHIFT_CONFIG_ROOT:-}"
	local DEFAULT_QUIET

	if [[ -z "${CONFIG_ROOT}" ]]; then
	    DEFAULT_QUIET=false
	else
	    DEFAULT_QUIET=true
	fi

    echo -n "existing logic: "
	if [[ "${OPENSHIFT_QUIET_OUTPUT:-${DEFAULT_QUIET}}" != "true" ]]; then
	    echo "true"
	else
	    echo "false"
	fi
}

function proposed_logic() {
    echo -n "proposed logic: "
    if [[ -n "${OPENSHIFT_QUIET_OUTPUT:-}" || ! -n "${OPENSHIFT_CONFIG_ROOT:-}" ]]; then
        echo "true"
    else
        echo "false"
    fi
}

echo "both unset"
existing_logic
proposed_logic

echo "config root unset, quiet output true"
OPENSHIFT_QUIET_OUTPUT=true existing_logic
OPENSHIFT_QUIET_OUTPUT=true proposed_logic

echo "config root unset, quiet output false"
OPENSHIFT_QUIET_OUTPUT=false existing_logic
OPENSHIFT_QUIET_OUTPUT=false proposed_logic

echo "config root set, quiet output unset"
OPENSHIFT_CONFIG_ROOT='/some/path' existing_logic
OPENSHIFT_CONFIG_ROOT='/some/path' proposed_logic

echo "config root set, quiet output true"
OPENSHIFT_CONFIG_ROOT='/some/path' OPENSHIFT_QUIET_OUTPUT=true existing_logic
OPENSHIFT_CONFIG_ROOT='/some/path' OPENSHIFT_QUIET_OUTPUT=true proposed_logic

echo "config root set, quiet output false"
OPENSHIFT_CONFIG_ROOT='/some/path' OPENSHIFT_QUIET_OUTPUT=false existing_logic
OPENSHIFT_CONFIG_ROOT='/some/path' OPENSHIFT_QUIET_OUTPUT=false proposed_logic
```

The output is:
```
both unset
existing logic: true
proposed logic: true
config root unset, quiet output true
existing logic: false
proposed logic: true
config root unset, quiet output false
existing logic: true
proposed logic: true
config root set, quiet output unset
existing logic: false
proposed logic: false
config root set, quiet output true
existing logic: false
proposed logic: true
config root set, quiet output false
existing logic: true
proposed logic: true
```